### PR TITLE
SLING-10482 Make it possible to validate values with property placeholders

### PR DESCRIPTION
**Pull request by bosschaert**

### CODE REVIEW

File: src/main/java/org/apache/sling/feature/extension/apiregions/api/config/PropertyDescription.java [hunk 1]
```diff
@@ -457,4 +465,33 @@ public PlaceholderPolicy getPlaceholderPolicy() {
     public void setPlaceholderPolicy(final PlaceholderPolicy policy) {
         this.placeholderPolicy = policy == null ? PlaceholderPolicy.DEFAULT : policy;
     }
+
+	/**
+	 * Get the placeholder regex
+	 * @return the placeholder regex or {@code null}
+	 */
+    public String getPlaceholderRegex() {
```
Comment (cziegeler) [inline src/main/java/org/apache/sling/feature/extension/apiregions/api/config/PropertyDescription.java:473]: Please add a @since tag for the new methods

**Review by cziegeler — APPROVED** [review]
Lgtm. thanks

File: src/main/java/org/apache/sling/feature/extension/apiregions/api/config/PropertyDescription.java (hunk 1 above)
Comment (bosschaert) [reply src/main/java/org/apache/sling/feature/extension/apiregions/api/config/PropertyDescription.java:474]: Thanks for the suggestion, @cziegeler - I've done this in my latest commit.

**Comment by cziegeler** [comment]
@bosschaert  Thanks, looks good to me

**Pull request merged**